arctic/arctic.py

```diff
@@ -202,6 +203,42 @@ def list_libraries(self):
                         libs.append(coll[:-1 * len(self.METADATA_COLL) - 1])
         return libs
 
+    @mongo_retry
+    def invalidate_libraries_cache(self):
+        """
+        Deletes the data in the collection used to cache the result
+        of list_libraries
+        :return:
+        """
+        self._conn.cache_db.cached_lib_names.drop()
```

Comment:
I presume dropping a collection is heavyweight, which might go against the idea of a cache here.

Reply:
So my mongo knowledge is limited but I changed my remove({}) to drop based on https://stackoverflow.com/questions/12147783/mongodb-trade-offs-of-dropping-a-collection-vs-removing-all-of-its-documents/12148106
Although I guess in a 1 element db remove might be cheaper.